### DISCUSSION TITLE
WAVEWATCH III Error in W3_grid

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#890] [Q&A] [answered]
Hello, my name is Federico Haspert and I am a beginner in the use of WW3. I am getting the following error message when running ww3_grid: " *** WAVEWATCH-III ERROR IN W3GRID : SOURCE TERMS REQUESTED BUT NOT SELECTED". It may be an error associated with the switches?. I hope you can help me.

**Comment by JessicaMeixner-NOAA** [comment, marked as answer]
What switch file did you use?

**Reply by fed1942** [reply]
I don't know because i didn't install the program (the person in question is on vacation). I think the line of code where the error occurs is: "&MISC FLAGTR = 4 /" (this is the only Namelist i use).

I just want to know if the error is associated with the switches or is there another problem.

Thank you in advance.

**Reply by JessicaMeixner-NOAA** [reply]
Unfortunately, you're likely going to have to recompile or get some information from the person on vacation.  Here's a section of the input file that might be worth looking into: https://github.com/NOAA-EMC/WW3/blob/develop/model/inp/ww3_grid.inp#L17-L23
